zipline/src/commonMain/kotlin/app/cash/zipline/internal/bridge/Endpoint.kt

```diff
@@ -233,4 +233,6 @@ internal class Endpoint internal constructor(
     open fun callEnd(call: Call, result: CallResult, startValue: Any?) {
     }
   }
+
+  internal companion object
```

Comment:
Some of the generate code attempts to access `Endpoint`'s companion object. Need to figure out why

Reply:
This'll be an issue when we start compiling Zipline itself with K2. Until then we don't need this.